app/models/post.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class Post < ApplicationRecord
-  belongs_to :user
+  belongs_to :author
```

Comment:
- [ ] There is no model named `Author`, so we need to specify the model being referred to here. Based on the feedback from the previous reviewer kindly add the `class_name` attribute.

  ```
  belongs_to :author, class_name: 'User'
  ```